### PR TITLE
More documentation

### PR DESCRIPTION
**Pull request by frewsxcv**
Rebase of #3 to work with stable and beta. I couldn't get `cfg_attr!()` to work correctly with conditionally turning on features, so I just disabled the nightly-only tests for now.

Fixes #3.

[Review on Reviewable](https://reviewable.io/reviews/servo/rust-fnv/4)

### CODE REVIEW

**Comment by SimonSapin** [comment]
r=me the version number bumped to 1.0.1 to publish on crates.io, where Servo can pick it up in order to have the docs show up on doc.servo.org.

**Comment by frewsxcv** [comment]
@SimonSapin 

1. it appears homu is not setup to manage this repo (https://github.com/servo/saltfs/issues/201)
2. it appears servo org members do not have write access to this repo, so i don't see the green merge button

**Comment by jdm** [comment]
I fixed the repo collaborators to address point 2.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,78 @@
...
+small—a perfect use case for FNV.
+
+
+## Usage
+
+To include this crate in your program, add the following to your `Cargo.toml`:
+
+```toml
+[dependencies]
+fnv = "1.0.0"
```
Comment (KiChjang) [inline README.md]: This should be bumped to 1.0.1

Comment (frewsxcv) [inline README.md]: I don't see any reason to bump this everytime we release a new `1.0.x` release

**Comment by SimonSapin** [comment]
Thanks @frewsxcv and @ogham !

**Pull request merged**